### PR TITLE
Savingfeature

### PR DESCRIPTION
**Pull request by Harish110**
loading csv into game 
restoring gameplay done

### CODE REVIEW

File: SimpCity/Game.cs [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
 
 namespace SimpCity {
     public enum BuildingTypes {
         Beach
     }
 
+
```
Comment (CaeNGnp) [inline SimpCity/Game.cs:14]: unintended newline?

File: SimpCity/Game.cs [hunk 2]
```diff
@@ -98,15 +103,53 @@ public class Game {
...
             // TODO: US-5 - restore state
-            throw new NotImplementedException();
+            // grid = filesaver.load()
+            // test
         }
 
-        public void Play() {
+
+            public void Play() {
```
Comment (CaeNGnp) [inline SimpCity/Game.cs:152]: unintended indentation?

File: SimpCity/buildings/House.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿namespace SimpCity.buildings {
+    class House : CityGridBuilding {
+        public static string Name { get; } = "House";
+        /// <summary>
+        /// The 3-character code for the building
+        /// </summary>
+        public static string Code { get; } = "HSE";
+        public House(BuildingInfo info) : base(info) { }
+    }
+}
```
Comment (CaeNGnp) [inline SimpCity/buildings/House.cs:10]: As agreed with tech lead @jordie007, only Beach will ship in Sprint 1 while the rest will go with Sprint 2. Should remove this

File: SimpCity/SimpCity.csproj [hunk 4]
```diff
@@ -44,6 +44,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="buildings\Beach.cs" />
+    <Compile Include="buildings\House.cs" />
```
Comment (CaeNGnp) [inline SimpCity/SimpCity.csproj:47]: ditto

**Review by CaeNGnp — CHANGES_REQUESTED** [review]
Almost looks good to me, good job. Just some minor nitpicks and it should be good to go in.

**Review by JSQJ — APPROVED** [review]
Code works fine. Able to load game

File: SimpCity/Game.cs (hunk 1 above)
Comment (Harish110) [reply SimpCity/Game.cs:14]: yes

File: SimpCity/buildings/House.cs (hunk 3 above)
Comment (Harish110) [reply SimpCity/buildings/House.cs:10]: ok will delete it

**Review by jordie007 — APPROVED** [review]
Code looks good to go. Pull request approved

**Pull request merged**